app/models/physical_storage.rb

```diff
@@ -10,6 +10,7 @@ class PhysicalStorage < ApplicationRecord
   belongs_to :physical_chassis, :inverse_of => :physical_storages
 
   has_many :storage_resources, :dependent => :destroy
+  has_many :host_initiators, :dependent => :destroy
```

Comment:
You want to throw in `has_many :san_addresses, :through => :host_initiators` ?

Reply:
The physical-storage is expected to have san_addresses of its own. The real addresses through which it actually connects to the SAN. We'll use the name `san_addresses` on physical-storage when we start dealing with these.
The set of all the san_addresses configured in a physical-storage's host_initiators could possibly be interesting when we start dealing with connecting actual computer-systems to consume the storage. I think an appropriate name could be `has_many registered_host_addresses`, so it's not confused with the physical-storages actual san-addresses.

Reply:
@gregoryb-at-ibm @agrare I changed it to `has_many :registered_host_addresses, :through => :host_initiators, :source => :san_addresses`, is it ok with u?

Reply:
done